crates/holochain_sqlite/src/db/access.rs

```diff
@@ -244,7 +244,7 @@ impl<Kind: DbKindT + Send + Sync + 'static> DbWrite<Kind> {
     ) -> DatabaseResult<Self> {
         let path = match path_prefix {
             Some(path_prefix) => {
-                let path = path_prefix.join(kind.filename());
+                let path = path_prefix.canonicalize()?.join(kind.filename());
```

Comment:
Rust canonicalize [is currently a no-no](https://github.com/rust-lang/rust/issues/42869) you can use [dunce::canonicalize](https://crates.io/crates/dunce).

Reply:
Grim, it sounds like canonicalize is the wrong tool for the job...

Reply:
Okay, better solution proposed that doesn't involve trying to match what sqlite has done. Just take the path it tells us it's using.